plugins/base/src/main/kotlin/translators/psi/DefaultPsiToDocumentableTranslator.kt

```diff
@@ -317,10 +317,19 @@ class DefaultPsiToDocumentableTranslator(
                     is PsiClassReferenceType -> {
                         val resolved: PsiClass = type.resolve()
                             ?: return UnresolvedBound(type.presentableText)
-                        if (resolved.qualifiedName == "java.lang.Object") {
-                            JavaObject
-                        } else {
-                            TypeConstructor(DRI.from(resolved), type.parameters.map { getProjection(it) })
+                        when {
+                            resolved.qualifiedName == "java.lang.Object" -> {
```

Comment:
We don't need brackets here, each time we return a simple instance

Reply:
ok